### PR TITLE
[RPS-458] Date format change for schedule publication

### PR DESCRIPTION
**Pull request by ngta1**
When scheduling a publication, the date format is now changed
from default format of M/d/yyyy to d/M/yyyy.

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/site/facets.feature [hunk 1]
```diff
@@ -136,6 +136,6 @@ Feature: Faceted search
         Given I navigate to the "search" page
         Then I should see the list with title "PUBLICATION STATUS" containing:
             | Published ( ... |
-            | Upcoming (2) |
+            | Upcoming (3) |
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/resources/features/site/facets.feature]: this might break on travis :D

Comment (wojtek-oledzki) [reply acceptance-tests/src/test/resources/features/site/facets.feature]: you need to clean-up after the "scheduled publication date format" test, to make test.cms + test.site pass :)

Comment (ngta1) [reply acceptance-tests/src/test/resources/features/site/facets.feature]: good spot.

File: acceptance-tests/src/test/resources/features/cms/createAndPublish.feature [hunk 2]
```diff
@@ -56,3 +56,15 @@ Feature: As am author I need to create a new publication
         When I click the "Add new about document..." menu option on the "Corporate Website/About" folder
         # No document options as you should automatically be creating an about type
         Then I should see no document options
+
+    @DeleteAfter
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/resources/features/cms/createAndPublish.feature]: :+1: simples

**Pull request merged**